tests/terraform_compliance/steps/its_key_value_test_cases/nested__dict_k.json

```diff
@@ -0,0 +1,30 @@
+{
+    "stash": [
+        {
+            "id": "a",
+            "values": [
+                {
+                    "id": "a.a",
+                    "tags": {
+                        "foo": "..."
+                    }
+                },
+                {
+                    "id": "a.b",
+                    "tags": {
+                        "not_foo": "..."
+                    }
+                },
+                {
+                    "id": "a.c",
+                    "tags": {
+                        "foo": "...",
+                        "not_foo": "..."
+                    }
+                }
+            ]
+        }
+    ],
+    "args": ["tags", "foo"],
+    "expected_ids": []
```

Comment:
We don't support dictionaries inside nested objects, should we?

Reply:
Hmm this is a tricky one. Thanks for pointing this out.

In theory, we should support it, but I believe in your specific case for this step, we can assume the current step is valid as long it passes all tests. Terraform (providers fault not terraform’s) can be quite unpredictable sometimes